docs/layouts/partials/toc.html

```
@@ -0,0 +1,48 @@
+{{ if not .Params.notoc -}}
+
+  {{ if eq .Type "models" -}}
+    {{/*
+      Model pages are built entirely from front matter, not markdown
+      headings, so .TableOfContents is always empty here. Same nav
+      Meshery already builds below, just rendered in the shared
+      sidebar slot instead of inline in the page body. See #20339.
+    */ -}}
+    <div class="td-toc-title">
+      <span class="td-toc-title__text">{{ i18n "toc_on_this_page" }}</span>
+      <a class="td-toc-title__link" title="{{ i18n "toc_top_of_page" }}" href="#"></a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)